pkg/release/release.go

```diff
@@ -299,3 +299,64 @@ func GetOCIManifest(tarPath string) (*ocispec.Manifest, error) {
 	// Return the embedded v1 manifest wrapped in the container/image struct
 	return &ociman.Manifest, err
 }
+
+// CopyBinaries takes the provided `rootPath` and copies the binaries sorted by
+// their platform into the pre-defined `$PWD/bin/$PLATFORM` directories.
+func CopyBinaries(rootPath string) error {
+	platformsPath := filepath.Join(rootPath, "client")
+	platforms, err := ioutil.ReadDir(platformsPath)
+	if err != nil {
+		return errors.Wrapf(err, "retrieve platforms from %s", platformsPath)
+	}
+
+	cwd, err := os.Getwd()
+	if err != nil {
+		return errors.Wrap(err, "get current working directory")
+	}
+
+	for _, platform := range platforms {
```

Comment:
This is the second loop around platforms I've seen in the code that is based on the filesystem tree. I'm wondering if at some point we should haver a known list of platform (os+arch) to be sure we are still running on a complete release.

Reply:
I was wondering that too, I think we do not change the platforms (linux, windows, darwin) very often so we probably could have them as pre-defined values.